vertical-pod-autoscaler/docs/features.md

```diff
@@ -54,6 +54,22 @@ To enable this feature, set the --round-cpu-millicores flag when running the VPA
 --round-cpu-millicores=50
 ```
 
+## Memory Recommendation Rounding
```

Comment:
Can this be added to the contents on line 6?
(side note: we should automate this TOC generation)